test/pdfs/.gitignore

```diff
@@ -36,6 +36,7 @@
 !issue7115.pdf
 !issue7180.pdf
 !issue7769.pdf
+!issue13130.pdf
```

Comment:
This ought to be a `link`ed test-case instead, since it's not a reduced one (and the copyright situation is unclear).